Helper/Webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Klaviyo\Reclaim\Helper;
+
+use Exception;
+use \Klaviyo\Reclaim\Helper\ScopeSetting;
+
+class Webhook extends \Magento\Framework\App\Helper\AbstractHelper
```

Comment:
should this be named Webhook or should it be more descriptive?

Reply:
like `KlaviyoWebhook` or something even more descriptive?